circle.yml

```diff
@@ -19,9 +21,9 @@ deployment:
       - git config --global user.name "CircleCI"
       - git config --global push.default simple
       - node scripts/publish_web.js
-      - mv npmrc-env .npmrc
```

Comment:
Why not just keep this? Then you don't need to have the `.npmrc` in the repo normally.